src/ssl.jl

```diff
@@ -598,14 +600,18 @@ function Base.eof(ssl::SSLStream)::Bool
             end
             # if we're here, we know there are unprocessed bytes,
             # so we call peek to force processing
-            @geterror ccall(
-                (:SSL_peek, libssl),
-                Cint,
-                (SSL, Ref{UInt8}, Cint),
-                ssl.ssl,
-                PEEK_REF,
```

Comment:
can the global `PEEK_REF` on L582 be removed now?

Reply:
yep, good call